_pages/en/product-fw-guidance.md

```diff
@@ -0,0 +1,352 @@
+
+Please direct enquiries about this Pilot directive to the IITB IT Strategy team.
+
+## Narrative
+
+This section provides context and describes the causal relationship between the elements of the Directive (rules to follow), its associated Standard (more detailed rules such as formats and specific information required), and their expected outcomes.
+Job Bank is used as an example.
+
+Employment and Social Development Canada (ESDC) provides multiples services via its [50 Programs](https://dialogue/grp/WG6278300/SitePages/Performance%20Information%20Profiles.aspx) to [improve the standard of living and quality of life for all Canadians](https://www.canada.ca/en/employment-social-development.html).
+One of those Programs is Job Bank.
+
+Products can be tied to Service delivery, then to Programs the following way.
+
+<figure>
+<a href="{{ site.baseurl }}/assets/images/product-service-mgt.png">
+        <img class="center" src="{{ site.baseurl }}/assets/images/product-service-mgt.png"
+        alt="This image depicts a the relationship between product management and service management. The image shows two rectangles: one labeled 'Product mgt' with the text Managing internal complexities to upkeep product(s) necessary for service delivery; and the other labeled 'Service mgt' with the text Facilitates outcomes that clients want to achieve, without them having to manage specific costs and risks. There's an arrow pointing from the Product mgt rectangle to the Service mgt rectangle labeled 'Enables' and with the text Services are based on one or more products. This image was altered from © αβ2ϲ and © AXELOS Limited 2019."
+        />
+  </a>
+    <figcaption>Figure 1. The Product - Service relationship</figcaption>
+</figure>
+
+A service consumer doesn't care about the costs and risks to operate the service.
+But a service leverages products for its delivery. Managing the internal complexities to sustain these products AND how they affect the service level is where product management fits.
+Moving to digital means some of those products will be exposed to service consumer (e.g. a website).
+
+The words “Job Bank” can actually be understood to mean three separate things:
+
+- It's a Program
+- It's a free Service to job seekers and employers, ultimately measured using the Program's [Performance Information Profile (PIP)](https://dialogue/grp/WG6278300/Program Inventory/Forms/AllItems.aspx?RootFolder=/grp/WG6278300/Program%20Inventory/Job%20Bank&FolderCTID=0x012000B9ED43B8F503194E9CF21B1ED35BB73E&View=%7b12E60D67-CC97-44CE-BBCA-B6E4EE0365E5%7d)
+- And it's a [family of products](https://014gc.sharepoint.com/sites/JobBankProductManagement/SitePages/products.aspx), most notably ten of them making up its website exposed and consumed by clients as part of Job Bank's service delivery (See [Appendix A](#appendix-a---business-view-of-job-banks-family-of-products) and [Appendix B](#appenix-b---it-view-of-job-banks-family-of-products-of-products)).

```

Comment:
The 'And' should be at the end of the text for the second bullet. But it should be in lower case. Each bullet to start with "It's"